README.md

```diff
@@ -1,94 +1,60 @@
 # OK
 
-**Elegant error handling in elixir pipelines. See [Handling Errors in Elixir](http://insights.workshop14.io/2015/10/18/handling-errors-in-elixir-no-one-say-monad.html) for a more detailed explanation**
+**Elegant error handling for Elixir. Built on the solid foundation of the result monad.**
 
-[Documentation for OK is available on hexdoc](https://hexdocs.pm/ok)
-
-## Installation
```

Comment:
Hmm, I agree it's complete boilerplate, but it's also very convenient to just copy/paste straight from the readme without another click. Ueberauth (and guardian), credo, mailgun, etc. Perhaps if we just condense it to the barest needed for the ultra lazy like myself, i.e. just the code block to copy?